packages/client/src/client/OpenFin/apps/LimitChecker/state.ts

```diff
@@ -40,7 +40,8 @@ const limitResult$ = limitCheckRequest$.pipe(
     limitBySymbol$(request.tradedCurrencyPair).pipe(
       take(1),
       map(([value, formattedValue]) => ({
-        notional: formattedValue,
+        notional: formatNotional(request.notional, ["k", "m"])[1],
```

Comment:
don't like this array dereferencing .. arbitrary "second value" .. but not a now thing